apps/web/pages/organizations/[organizationId]/components/invoiceTable.tsx

```diff
@@ -40,13 +40,25 @@ export const InvoiceTable = ({ invoices, organizationId }: InvoiceTableProps): J
             <tr key={invoice.id}>
               <td>{invoice.invoiceDate}</td>
               <td>{invoice.customerName}</td>
-              <td>{invoice.invoiceItems.reduce((sum, invoiceItem) => sum + invoiceItem.duration, 0)}</td>
               <td>
-                {invoice.invoiceItems.reduce(
-                  (sum, invoiceItem) => sum + invoiceItem.duration * invoiceItem.hourlyRate,
-                  0,
+                {(invoice.invoiceItems.reduce((sum, invoiceItem) => sum + invoiceItem.duration, 0) / 60).toLocaleString(
+                  navigator.languages,
+                  {
+                    minimumFractionDigits: 2,
+                    maximumFractionDigits: 2,
+                  },
                 )}
               </td>
+              <td>
```

Comment:
Could we also add the `Duration` column to give a full picture of the `Total amount`?
```suggestion
              </td>
              <td>
                {invoice.invoiceItems
                  .reduce((sum, invoiceItem) => sum + invoiceItem.duration, 0)
                  .toLocaleString(navigator.languages, {
                    minimumFractionDigits: 2,
                    maximumFractionDigits: 2,
                  })}
              </td>
              <td>
```

Reply:
I believe this *is* the duration column, just named differently. Unless you meant something else?

Reply:
What this table missing is the `Hourly rate`, which is tricky to implement, because different `Invoice item`s have different rate. So, we decided against it.

Reply:
Got it.